Fix up the docs to make configuration match screenshot

davidfmatheson opened the pull request:
The screenshot has blue background for numbers on non-current windows, and space between text and number on right. Update sample config to make sure it matches result in screenshot. Without `@thm_blue` it will default to grey.

kjnsn commented:
Yeah the screenshot needs updating. The colors were changed to improve accessibility. Would you mind updating the screenshot?

davidfmatheson commented:
I was updating the config to what I thought would generate the screenshot. You'd rather the screenshot be updated to match what was in the config without changes?

kjnsn commented:
Yes

davidfmatheson commented:
Ok, replaced screenshot, and removed the default peach/blue in config. I left the space after `#W` because that is necessary to make sure the text doesn't bump up against the right-hand numbers.

kjnsn commented:
Thank you so much!!
